notebooks/finetuning/Deploy your own finetuned command-r-0824.ipynb

```diff
@@ -22,19 +22,19 @@
     "        1. **aws-marketplace:ViewSubscriptions**\n",
     "        1. **aws-marketplace:Unsubscribe**\n",
     "        1. **aws-marketplace:Subscribe**  \n",
-    "    2. or your AWS account has a subscription to the packages for **cohere-command-R-v2-byoft**. If so, skip step: [Subscribe to the bring your own finetuning algorithm](#1.-Subscribe-to-the-bring-your-own-finetuning-algorithm)\n",
+    "    2. or your AWS account has a subscription to the packages for **cohere-command-R-v2-byoft**. If so, skip step: [Subscribe to the bring your own finetuning algorithm](#subscribe)\n",
```

Comment:
Please don't merge it until we confirm there are no more changes. Otherwise, we will need to submit many small PRs to make many small changes